client/my-sites/domains/domain-management/list/index.jsx

```diff
@@ -286,7 +286,7 @@ export class List extends React.Component {
 				className="domain-management-list__change-primary-button"
 				onClick={ this.enableChangePrimaryDomainMode }
 			>
-				{ this.props.translate( 'Change Primary', {
+				{ this.props.translate( 'Change primary', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.